package.json

```diff
@@ -65,81 +94,131 @@
-    "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
-    "start": "tt-serve",
-    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
-    "start-prod": "tt-serve --env.disableHotReloading -p",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
-    "test": "npm run lint && npm run jest && npm run wdio",
-    "wdio-default": "wdio",
-    "wdio-lowlight": "NODE_OPTIONS=--max-old-space-size=2048 THEME=clinical-lowlight-theme wdio",
-    "wdio-fusion": "THEME=orion-fusion-theme wdio",
-    "wdio": "npm run wdio-default && npm run wdio-lowlight && npm run wdio-fusion",
-    "version": "lerna version --no-git-tag-version && node ./scripts/changelog-updater/index.js"
+  "dependencies": {
+    "d3": "^5.16.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.5.0",
     "@babel/core": "^7.5.0",
     "@babel/plugin-proposal-object-rest-spread": "^7.5.0",
     "@babel/plugin-transform-object-assign": "^7.2.0",
     "@babel/plugin-transform-runtime": "^7.5.0",
-    "@babel/preset-env": "^7.5.0",
+    "@babel/polyfill": "^7.10.4",
+    "@babel/preset-env": "^7.10.4",
     "@babel/preset-react": "^7.0.0",
-    "@cerner/eslint-config-terra": "^4.0.0",
+    "@babel/runtime": "^7.10.4",
     "@cerner/browserslist-config-terra": "^3.0.0",
+    "@cerner/carbon-site-helpers": "^1.11.3",
+    "@cerner/eslint-config-terra": "^4.0.0",
+    "@cerner/svg-to-carbon": "^1.1.3",
+    "@semantic-release/git": "^9.0.0",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^24.8.0",
+    "babel-loader": "^8.1.0",
+    "babel-plugin-istanbul": "^6.0.0",
+    "babel-plugin-minify-replace": "^0.5.0",
+    "babel-preset-minify": "^0.5.1",
     "check-installed-dependencies": "^1.0.0",
+    "classnames": "^2.2.5",
+    "commander": "^5.1.0",
     "core-js": "^3.1.3",
+    "css-loader": "^3.6.0",
+    "cssnano": "^4.1.10",
     "danger": "^9.1.0",
+    "docdash": "^1.2.0",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "enzyme-to-json": "^3.2.2",
     "eslint": "^6.1.0",
+    "eslint-plugin-import": "^2.22.1",
+    "express": "^4.17.1",
+    "express-rate-limit": "^5.1.3",
     "glob": "^7.1.1",
+    "html-webpack-plugin": "^4.3.0",
+    "husky": "^4.3.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.8.0",
-    "lerna": "^3.16.4",
+    "jsdoc": "^3.6.4",
+    "karma": "^5.1.0",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-coverage": "^2.0.2",
+    "karma-jasmine": "^3.3.1",
+    "karma-parallel": "^0.3.1",
+    "karma-sourcemap-loader": "^0.3.7",
+    "karma-webpack": "^4.0.2",
+    "lerna": "^3.22.1",
+    "less": "^3.12.2",
+    "less-loader": "^7.0.2",
     "link-parent-bin": "^1.0.0",
+    "loader-utils": "^1.4.0",
+    "mini-css-extract-plugin": "^0.9.0",
+    "one-cerner-style-icons": "github:cerner/one-cerner-style-icons",
+    "postcss-loader": "^3.0.0",
+    "prettier": "^2.0.5",
+    "prop-types": "^15.5.8",
+    "puppeteer": "^5.3.1",
```

Comment:
What do you use puppeteer for?

Reply:
the karma config file imports puppeteer:
<img width="698" alt="CleanShot 2020-10-27 at 16 56 10@2x" src="https://user-images.githubusercontent.com/38024451/97366555-836b0000-1875-11eb-9db1-7ea8d71989ce.png">